src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -209,45 +309,10 @@ public void balanceCorrection(double gyroPitchAngle) {
     }
   }
 
-  public void distanceResetPID() {
-    /** This should be run when stopping a pid command. */
-    distanceControllerEnabled = false;
-  }
-
-  public void distanceSetGoal(double targetDistance) {
-    m_distanceController.setGoal(AverageDistance() + targetDistance);
-  }
-
-  private void calculateDistanceRate(double targetDistance) {
-    if (!distanceControllerEnabled) {
-      m_distanceController.reset(AverageDistance());
-      m_distanceController.setGoal(AverageDistance() + targetDistance);
-      distanceControllerEnabled = true;
-    }
-    distanceThrottleRate =
-        MathUtil.clamp(m_distanceController.calculate(AverageDistance()), -1.0, 1.0);
-  }
-
-  public void driveToDistance(double targetDistance) {
-    this.calculateDistanceRate(targetDistance);
-    double leftStickValue = turnRotateToAngleRate;
-    double rightStickValue = turnRotateToAngleRate;
-    if (!m_distanceController.atGoal()) {
-      this.tankDrive(leftStickValue, rightStickValue);
-    }
-  }
-
-  public void driveAndTurn(double gyroYawAngle, double TargetAngleDegrees, double targetDistance) {
-    /*
-    This lets you set a gyro angle and a distance you need to travel.
-    this should not be used in auto mode.
-     */
-    this.calcuateAngleRate(gyroYawAngle, TargetAngleDegrees);
-    this.calculateDistanceRate(targetDistance);
-    double leftStickValue = distanceThrottleRate + turnRotateToAngleRate;
-    double rightStickValue = distanceThrottleRate - turnRotateToAngleRate;
-    if (!m_distanceController.atGoal() || !m_turnController.atGoal()) {
-      this.tankDrive(leftStickValue, rightStickValue);
+  public void driveToRelativePosition(double targetPosition) {
```

Comment:
i removed the drive and turn, but this is in case we need to add it back  or do something similar as it allows us to easily interface with the position based pid on the neo motors.